supplementary_style_guide/glossary_terms_conventions/general_conventions/p.adoc

```diff
@@ -57,6 +57,17 @@
 
 *See also*:
 
+[[peer-pod]]
+==== image:images/yes.png[yes] peer pod (noun, adjective)
+*Description*: In Red{nbsp}Hat OpenShift sandboxed containers, a _peer pod_ extends the concept of a standard pod. For bare metal servers, workloads run as containers inside a standard pod on a virtual machine that is created on a worker node. For public clouds or environments that do not support nested virtualization, workloads run as containers inside a peer pod on a virtual machine that is created on a hypervisor, such as VMware, or a public cloud instance, such as Azure or AWS.
```

Comment:
```suggestion
*Description*: In Red{nbsp}Hat OpenShift sandboxed containers, a _peer pod_ extends the concept of a standard pod. For bare metal servers, workloads run as containers inside a standard pod on a virtual machine that is created on a worker node. For public clouds or environments that do not support nested virtualization, workloads run as containers inside a peer pod on a virtual machine that is created on a hypervisor, such as vSphere Hypervisor, or a public cloud instance, such as Azure or AWS.
```

Reply:
"VMware" is broad and extends beyond a hypervisor, so maybe something more specific like "vSphere Hypervisor"?

https://www.vmware.com/products/vsphere-hypervisor.html.html

Reply:
This is true. My focus was not to confuse writers that are unfamiliar with hypervisors and VMWare. Some writers might search a term and make assumptions. Maybe the exampels could be removed?